lib/routes/javdb/actors.ts

```diff
@@ -46,6 +46,5 @@ async function handler(ctx) {
     };
 
     const title = `JavDB${filters[filter] === '' ? '' : ` - ${filters[filter]}`} `;
-
```

Comment:
Please also update
https://github.com/DIYgod/RSSHub/blob/5e142390629cd8c0b1fc4a49fd6cc59b9acedce5/lib/routes/javdb/actors.ts#L10
to
```ts
    features: {
        requireConfig: [{
            name: 'JAVDB_SESSION',
            description: 'Description here',
            optional: true,
        }],
```

Reply:
Thank you for your guidance, I have made the changes, please review again.